config/environments/production.rb

```diff
@@ -136,15 +136,18 @@
   config.skylight.probes << "graphql"
 
   # Add throttling to application
+  require_relative '../../lib/rack_throttle/rules'
+
   rules = [
     { method: "POST", limit: ENV['GRAPH_API_RATE_LIMIT'] },
-    { method: "POST", path: "/graphql", limit: ENV['GRAPH_API_RATE_LIMIT'] }]
+    { method: "POST", path: "/graphql", limit: ENV['GRAPH_API_RATE_LIMIT'] },
```

Comment:
I checked the documentation for `rack-throttle` and it appears that the second line - `method: "POST", path: "/graphql"` might not be required, since that's covered by the more general rule `method: "POST", limit: ENV['GRAPH_API_RATE_LIMIT']`. Could  you confirm if the second rule specific to `path: '/graphql'` is required?

Reply:
Yes, @harigopal Tested it. It's redundant.